classifiers/reference_quality/text_length_classifier/config.py

```diff
@@ -0,0 +1,45 @@
+from util.configs import build_classifier_function_config
+from util.enums import State, RefineryDataType, SelectionType, BricksVariableType
+from . import text_length_classifier, INPUT_EXAMPLE
+
+
+def get_config():
+    return build_classifier_function_config(
+        function=text_length_classifier,
+        input_example=INPUT_EXAMPLE,
+        issue_id=348,
+        tabler_icon="RulerMeasure",
+        min_refinery_version="1.7.0",
+        state=State.PUBLIC.value,
+        type="python_function",
+        available_for=["refinery", "common"],
+        part_of_group=[
+            "reference_quality",
+            "text_analysis"
+        ],  # first entry should be parent directory
+        # mapping lables for cognition
+        cognition_init_mapping={
+            "short": "Needs fix",
+            "medium": "null",
+            "long": "null",
```

Comment:
this shouldn't be a null string but the value None

Reply:
not sure if it's possible though :D please test, if not ill think of something on the interpreter side

Reply:
Tested it and works fine. The attached picture shows how this is then stored in the CMS. This should be the same as doing a `json.dump(None)`, if I am not mistaken.

![cms](https://github.com/code-kern-ai/bricks/assets/125879388/85843677-13f8-4176-9efc-51da72896ed5)

Reply:
works for me